cmd/timoni/bundle_lint.go

```diff
@@ -96,16 +96,6 @@ func runBundleLintCmd(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	apiVersion := v.LookupPath(cue.ParsePath(apiv1.BundleAPIVersionSelector.String()))
```

Comment:
Is this removal an incidental change -- maybe it could go in its own commit, just to keep things tidy.

Reply:
The apiVersion is now being validated by the schema `apiVersion: string & =~"^v1alpha1$"` so there is no need to look it up in Go. If the apiVersion is not specified or if its value is not v1alpha1 linting will fail.